libbeat/tests/system/base.py

```diff
@@ -17,3 +21,115 @@ def setUpClass(self):
             "winlogbeat"
         ]
         super(BaseTest, self).setUpClass()
+
+    def clean(self):
```

Comment:
This clean method is about clearing ES templates and ilm policies. Really put it in BaseTest?